packages/node-core/src/indexer/storeCache/types.ts

```diff
@@ -66,3 +66,8 @@ export type SetValue<T> = {
 export type SetData<T> = Record<string, SetValueModel<T>>;
 
 export class GetData<T extends {}> extends LRUCache<string, T, unknown> {}
+
+export interface FileExporter {
```

Comment:
This might not be just for files, it could upload it somewhere or do various things that aren't files.

It would also be good to add some js doc comments to the methods